powerfactory/powerfactory-converter/src/main/java/com/powsybl/powerfactory/converter/TransformerConverter.java

```diff
@@ -178,8 +177,8 @@ void createThreeWindings(DataObject elmTr3) {
         Optional<PhaseAngleClockModel> pac3 = pac3WModel.getEnd(windingTypeEnds.get(2));
         if (pac2.isPresent() || pac3.isPresent()) {
             t3wt.newExtension(ThreeWindingsTransformerPhaseAngleClockAdder.class)
-                .withPhaseAngleClockLeg2(pac2.isPresent() ? pac2.get().pac : 0)
-                .withPhaseAngleClockLeg3(pac3.isPresent() ? pac3.get().pac : 0).add();
+                .withPhaseAngleClockLeg2(pac2.map(phaseAngleClockModel -> phaseAngleClockModel.pac).orElse(0))
+                .withPhaseAngleClockLeg3(pac3.map(angleClockModel -> angleClockModel.pac).orElse(0)).add();
```

Comment:
Can you use the same variable name for both lambda? (And maybe a shorter one, as `model`?)